scripts/test-infra/verify.sh

```diff
@@ -17,8 +17,7 @@ curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring
 curl -Os https://uploader.codecov.io/latest/linux/codecov
 chmod +x codecov
 
-# TODO: re-enable logcheck when https://github.com/kubernetes-sigs/logtools/pull/31 is merged
-#go install sigs.k8s.io/logtools/logcheck@v0.8.1
+go install sigs.k8s.io/logtools/logcheck@v0.9.1-0.20251007102500-d35c84c015fe
```

Comment:
maybe keep the NOTE to update it to the actual release, or let's not forget it. 
LGTM